app/code/core/Mage/Adminhtml/controllers/Sales/OrderController.php

```diff
@@ -136,6 +136,8 @@ public function emailAction()
     {
         if ($order = $this->_initOrder()) {
             try {
+                // FIX. Enable re-sending again - http://magento.stackexchange.com/a/37690/78
+				$order->setEmailSent(false);
```

Comment:
Preferred way to fix is while on your PR branch just replace the tabs with spaces, use `git commit -a --amend` to fix the existing commit and then `git push -f` to force-push over the old commit.